x-pack/metricbeat/module/aws/ec2/_meta/fields.yml

```diff
@@ -5,55 +5,55 @@
   release: ga
   fields:
     - name: cpu.total.pct
-      type: scaled_float
+      type: float
```

Comment:
Why is scaled float not enough for a percentage value?

Reply:
@ruflin Sorry for some reason I thought float is more efficient than scaled_float... I just checked and it's the opposite... `For floating-point types, it is often more efficient to store floating-point data into an integer using a scaling factor, which is what the scaled_float type does under the hood`. I will close this PR.